source/patterns/00-atoms/doi/doi.twig

```diff
@@ -0,0 +1,7 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['doi']) }) -%}
+
+<div {{- html.attributes(attributes|default({})) }}>
+  <a href="https://doi.org/{{doi}}" class="doi__link">https://doi.org/{{doi}}</a>
```

Comment:
Could do with a `with` and `block` wrapping an include of the `link` pattern here, rather than hard coding the link directly.